src/components/__tests__/SortPicker.ts

```diff
@@ -380,3 +380,100 @@ describe('Created to Updated', () => {
     expect(thoughtValues).toMatchObject(['a', 'c', 'd'])
   })
 })
+
+it('home: Note Asc', async () => {
+  act(() => {
+    store.dispatch([
+      importText({
+        text: `
+          - a
+            - =note
+              - 2
+          - b
+            - =note
+              - 3
+          - c
+            - =note
+              - 1
+        `,
+      }),
+      setCursor(['a']),
+    ])
+  })
+
+  await act(vi.runOnlyPendingTimersAsync)
+
+  await click('[data-testid="toolbar-icon"][aria-label="SortPicker"]')
+  await click('[aria-label="sort options"] [aria-label="Note"]')
+  await act(() => vi.runAllTimersAsync())
+
+  const thoughts = screen.getAllByTestId(/thought/)
+  expect(thoughts.map((child: HTMLElement) => child.textContent)).toMatchObject(['c1', 'a2', 'b3'])
+})
+
+it('home: Note Desc', async () => {
+  act(() => {
+    store.dispatch([
+      importText({
+        text: `
+          - =sort
+            - Note
+          - a
+            - =note
+              - 2
+          - b
+            - =note
+              - 3
+          - c
+            - =note
+              - 1
+        `,
+      }),
+      setCursor(['a']),
+    ])
+  })
+
+  await act(vi.runOnlyPendingTimersAsync)
+
+  await click('[data-testid="toolbar-icon"][aria-label="SortPicker"]')
+  await click('[aria-label="sort options"] [aria-label="Note"]')
+  await act(() => vi.runAllTimersAsync())
+
+  const thoughts = screen.getAllByTestId(/thought/)
+  expect(thoughts.map((child: HTMLElement) => child.textContent)).toMatchObject(['b3', 'a2', 'c1'])
+})
+
+it('home: Note Desc with edit', async () => {
+  act(() => {
+    store.dispatch([
+      importText({
+        text: `
+          - =sort
+            - Note
+          - c
+            - =note
+              - 1
+          - a
+            - =note
+              - 2
+          - b
+            - =note
+              - 3
+        `,
+      }),
+      setCursor(['a']),
+    ])
+  })
+
+  await act(vi.runOnlyPendingTimersAsync)
+
+  // Edit the note value of 'a' from 2 to 4
+  act(() => {
+    store.dispatch([editThoughtByContext(['a', '=note', '2'], '4')])
+  })
```

Comment:
It would be better to use a more user-centric way of interacting with the note rather than dispatching to the global store directly.

See: https://github.com/karunkop/em/blob/d82e40db37283cae9a9dad871c9a2991e6026fc0/src/components/__tests__/Note.ts#L378-L385

We currently dispatch `importText` directly to the store at the beginning of tests for simplicity, but [soon this will be refactored](https://github.com/cybersemics/em/issues/2980) to use `paste`, which is more user-centric and hides the global store access.